### PR TITLE
fix(installer): Fix dockerhub rate limiting in E2E tests

### PR DESCRIPTION
**Pull request by BaptisteFoy**
### What does this PR do?
Fixes installer E2E tests that were skipped due to dockerhub rate limiting

### Motivation
Working E2E

### Additional Notes
N/A

### Possible Drawbacks / Trade-offs
N/A

### Describe how to test/QA your changes
Just tests passing :)

### CODE REVIEW

File: test/new-e2e/tests/updater/linux_test.go [hunk 1]
```diff
@@ -340,8 +340,10 @@ func (v *vmUpdaterSuite) TestPurgeAndInstallAPMInjector() {
 	require.NotNil(v.T(), err)
 	_, err = host.Execute(`grep "/opt/datadog-packages/datadog-apm-inject" /etc/docker/daemon.json`)
 	require.NotNil(v.T(), err)
-	_, err = host.Execute(`test -f /etc/docker/daemon.json.bak`)
+	_, err = host.Execute(`test -f /etc/docker/daemxn.json.bak`)
```
Comment (paullegranddc) [inline test/new-e2e/tests/updater/linux_test.go:343]: Is this deliberate?

Comment (BaptisteFoy) [reply test/new-e2e/tests/updater/linux_test.go]: Oops, good catch. That test should've been removed anyway, we're not backing up the daemon config anymore.

Comment (BaptisteFoy) [reply test/new-e2e/tests/updater/linux_test.go]: Fixed in [71d47c1](https://github.com/DataDog/datadog-agent/pull/24676/commits/71d47c1410f44120d9dc522ec72ee1113e59bb3c)

**Review by coignetp — APPROVED** [review]

**Review by chouquette — APPROVED** [review]

**Pull request merged**